rules/windows/execution_unusual_dns_service_children.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = "An unexpected process spawned from dns.exe, which is the process responsible for DNS server services."
+false_positives = [
+    """
+    Werfault.exe can spawn in a legitimate manner when dns.exe crashes but this is very unlikely, but it can also spawn
+    as a result Denial of Service (DoS) attempts to intentionally crash the service.
+    """,
```

Comment:
```suggestion
false_positives = [
    """
    Werfault.exe can spawn in a legitimate manner when dns.exe crashes, but this is very unlikely.  It can also spawn
    as a result Denial of Service (DoS) attempts to intentionally crash the service.
    """,
```